orchagent/fdborch.cpp

```diff
@@ -476,7 +558,7 @@ void FdbOrch::updateVlanMember(const VlanMemberUpdate& update)
     }
 }
 
-bool FdbOrch::addFdbEntry(const FdbEntry& entry, const string& port_name, const string& type)
+bool FdbOrch::addFdbEntry(FdbEntry& entry, const string& port_name, const string& type)
```

Comment:
Yes, thats indeed what I thought before removing the const keyword. Then we would ask, why even pass port_name as a separate argument to this function when its already part of entry now. Answer is, when we learn any FDB entry before port is created, we would save these FDB entries with that port name as the key. Later when the port is created and added to VLAN, updateVlanMember() would call addFdbEntry for all the saved FDB entries. Yes, we can still make it happen, but takes more work. Hence thought of doing that cleanup later. The correct cleanup is,  we should change the prototype of addFdbEntry() itself by accepting only entry as parameter. That is just like port_name we should make "type" also part of the entry, because at several other places we need to distinguish between static and dynamic entries and its all commented out as TBD.

Anyways for now, I will keep the entry constant and remove port_name from function prototype.